src/test/java/org/apache/drillui/test/framework/steps/AuthSteps.java

```diff
@@ -14,17 +14,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package steps;
+package org.apache.drillui.test.framework.steps;
 
-import pages.BasePage;
-import pages.NavigationPage;
+import org.apache.drillui.test.framework.initial.TestProperties;
+import org.apache.drillui.test.framework.initial.WebBrowser;
+import org.apache.drillui.test.framework.pages.BasePage;
+import org.apache.drillui.test.framework.pages.NavigationPage;
 
 public final class AuthSteps {
 
   private AuthSteps() {
   }
 
   public static NavigationPage login(String login, String password) {
+    WebBrowser.getDriver().get(TestProperties.drillHost);
```

Comment:
Please remove, we don't need to return to the base page in every step.

Reply:
Done. We go to the Drill main page only if framework successfully gets error text. See class ErrorPage